setup.py

```diff
@@ -179,10 +180,17 @@ def run(self):
     author=__author__,
     author_email='info@ansible.com',
     url='https://ansible.com/',
+    project_urls={
+        'Bug Tracker': 'https://github.com/ansible/ansible/issues',
+        'Documentation': 'http://docs.ansible.com/ansible/',
```

Comment:
Use `https` instead of `http`. Yes, the site currently redirects back to the HTTP version, but that should eventually be fixed.